eng/native/naming.props

```diff
@@ -18,7 +18,8 @@
         <LibPrefix>lib</LibPrefix>
         <LibSuffix>.dylib</LibSuffix>
         <StaticLibSuffix>.a</StaticLibSuffix>
-        <SymbolsSuffix>.dwarf</SymbolsSuffix>
+        <!-- <SymbolsSuffix>.dwarf</SymbolsSuffix> -->
+        <SymbolsSuffix>.dSYM</SymbolsSuffix>
```

Comment:
This might quickly become a leaky abstraction. For example, we also are missing updating `GetSymbolPackageFiles` in https://github.com/dotnet/runtime/blob/main/src/installer/pkg/projects/Directory.Build.targets (the excludes and all that)